62_CAD/shaders/globals.hlsl

```diff
@@ -265,6 +271,12 @@ NBL_CONSTEXPR float InvalidStyleStretchValue = nbl::hlsl::numeric_limits<float>:
 // TODO[Przemek]: we will need something similar to LineStyles but related to heigh shading settings which is user customizable (like LineStyle stipple patterns) and requires upper_bound to figure out the color based on height value.
 // We'll discuss that later or what it will be looking like and how it's gonna get passed to our shaders.
 
+struct TriangleMeshVertex
```

Comment:
If this is your vertex data you pull in vertex buffer,  it needs to be in double precision.
+ no need for this struct, just use a `pfloat64_t3` the z value is height

Reply:
does it matter when its this struct or `pfloat64_t3`?

Reply:
if performance wise it doesn't matter then i would prefer to keep the struct

Reply:
it makes code a little bit more self explanatory